setup.py

```diff
@@ -27,6 +27,7 @@
 
 # Allow manually disabling seccomp on old kernels. WSL doesn't have seccomp.
 has_seccomp = not is_wsl and os.environ.get('DMOJ_USE_SECCOMP') != 'no'
+has_pcre = os.environ.get('DMOJ_USE_PCRE') != 'no'
```

Comment:
If this is made optional, it should be added to the appropriate `README.md` section.